ci-operator/step-registry/ipi/gcp/ovn/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- abhinavdahiya
```

Comment:
I don't believe he's at RH anymore, we need different owners for this.

Reply:
any idea who would be a good fit? ideally someone who works with ipi for gcp

Reply:
Maybe the ovn team?  Like https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/gcp/ovn/OWNERS

Reply:
yup, makes sense to me. I used that owners file and updated the registry json